auditors/image/image.go

```diff
@@ -12,8 +12,11 @@ import (
 const Name = "image"
 
 const (
+	// ImageTagMissing occurs when the container image tag is missing
 	ImageTagMissing   = "ImageTagMissing"
+	// ImageTagIncorrect occurs when the container image tag does not match the one registered in podspec
```

Comment:
```suggestion
	// ImageTagIncorrect occurs when the container image tag does not match the user-provided value
```

Context: https://github.com/Shopify/kubeaudit/blob/master/docs/auditors/image.md#examples